Assets/MixedRealityToolkit/_Core/Devices/BaseController.cs

```diff
@@ -84,7 +84,7 @@ protected IMixedRealityInputSystem InputSystem
         /// <param name="controllerType"></param>
         public void SetupConfiguration(Type controllerType)
         {
-            if (MixedRealityManager.Instance.ActiveProfile.EnableControllerMapping)
+            if (MixedRealityManager.Instance.ActiveProfile.IsControllerMappingEnabled)
```

Comment:
Needs to also have a check for "MRManager.HasActiveProfile"

Should this be a standard?

Reply:
Still trying to figure that out.

Reply:
I don't think we'll need to call `HasActiveProfile` in classes we know is created by a profile